monitoring/Monitoring-batch-reference-data-example.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6c0cd3c0",
+   "metadata": {},
+   "source": [
+    "# 0. Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "859e247f",
+   "metadata": {},
+   "source": [
+    "### 0.1 Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf9395b9",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from datetime import datetime\n",
+    "from random import randint, uniform\n",
+    "import time\n",
+    "from typing import List\n",
+    "\n",
+    "try:\n",
+    "    import numpy as np\n",
+    "except ImportError:\n",
+    "    !pip install numpy\n",
+    "    import numpy as np\n",
+    "\n",
+    "try:\n",
+    "    import pandas as pd\n",
+    "except ImportError:\n",
+    "    !pip install pandas\n",
+    "    import pandas as pd\n",
+    "\n",
+    "try:\n",
+    "    from verta import Client, environment\n",
+    "    from verta.dataset import Path\n",
+    "    from verta.dataset.entities import Dataset\n",
+    "    from verta.deployment._deployedmodel import DeployedModel\n",
+    "    from verta.endpoint import Endpoint\n",
+    "    from verta.registry import VertaModelBase\n",
+    "    from verta.registry.entities import RegisteredModel, RegisteredModelVersion\n",
+    "    from verta.tracking.entities import ExperimentRun\n",
+    "    from verta.utils import ModelAPI\n",
+    "except ImportError:\n",
+    "    !pip install verta\n",
+    "    from verta.dataset import Path\n",
+    "    from verta.dataset.entities import Dataset\n",
+    "    from verta.deployment._deployedmodel import DeployedModel\n",
+    "    from verta.endpoint import Endpoint\n",
+    "    from verta.registry import VertaModelBase\n",
+    "    from verta.registry.entities import RegisteredModel, RegisteredModelVersion\n",
+    "    from verta.tracking.entities import ExperimentRun\n",
+    "    from verta.utils import ModelAPI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d780235d",
+   "metadata": {},
+   "source": [
+    "### 0.2 Verta Client Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ee583ea0",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Use local env vars or uncomment and fill out the lines below:\n",
+    "os.environ['VERTA_EMAIL'] = 'ewagner@verta.ai'\n",
+    "os.environ['VERTA_DEV_KEY'] = 'fb0c7d6c-8f91-4f87-8730-febd36f74553'\n",
+    "os.environ['VERTA_HOST'] = 'ewagner2.dev.verta.ai'\n",
```

Comment:
never commit these

Reply:
Forgot to strip that out before committing.  My bad.  I have removed that key and generated a new one.